status/discover.go

```diff
@@ -88,119 +91,110 @@ func getPathURL(endpoint string) (path string) {
 	return u.Path
 }
 
-func checkTracingService() (url string, err error) {
-	conf := config.Get()
-	tracing := config.TracingDefaultService
-	jaeger := config.JaegerQueryDefaultService
-	ns := config.IstioDefaultNamespace
-	service := tracing
+func checkTracingService() (url string) {
+	tracingConfig := config.Get().ExternalServices.Tracing
+	ns := config.IstioDefaultNamespace // TODO: don't use default!
```

Comment:
@aljesusg I didn't change it because I'm not 100% sure, but it seems wrong to me to use `IstioDefaultNamespace` instead of the configured istio namespace. WDYT? Shouldn't we use `config.IstioNamespace` instead?

Reply:
Yes, matt told me about we should use the namespace where is deployed the environment.
Here is the line that I saw and I failed ^^
```go
c.IstioNamespace = strings.TrimSpace(getDefaultString(EnvIstioNamespace, IstioDefaultNamespace))
```

so it should be like

```go
c.ExternalServices.Tracing.Namespace = strings.TrimSpace(getDefaultString(EnvTracingServiceNamespace, c.IstioNamespace))
```

instead

```go
c.ExternalServices.Tracing.Namespace = strings.TrimSpace(getDefaultString(EnvTracingServiceNamespace, IstioDefaultNamespace))
```

?

Reply:
Yes I guess, something like that.. is there already a JIRA about that? or I can change it from this PR

Reply:
Created https://issues.jboss.org/browse/KIALI-2963